Build with UCONTEXT everywhere for current glibc.  Patch pulled from
upstream stable branch, will be in 1.4.5.

--- openafs.orig/src/config/param.alpha_linux_26.h
+++ openafs/src/config/param.alpha_linux_26.h
@@ -81,6 +81,11 @@
 #define SYS_NAME_ID    SYS_NAME_ID_alpha_linux_26
 
 
+#ifdef __GLIBC__
+#if (__GLIBC__ > 2) || (__GLIBC__ == 2 && __GLIBC_MINOR__ > 3)
+#define USE_UCONTEXT
+#endif
+#endif
 #endif /* AFS_PARAM_H */
 				     
 #else /* !defined(UKERNEL) */
--- openafs.orig/src/config/param.parisc_linux24.h
+++ openafs/src/config/param.parisc_linux24.h
@@ -70,6 +70,11 @@
 #endif
 #endif /* KERNEL */
 
+#ifdef __GLIBC__
+#if (__GLIBC__ > 2) || (__GLIBC__ == 2 && __GLIBC_MINOR__ > 3)
+#define USE_UCONTEXT
+#endif
+#endif
 #endif /* AFS_PARAM_H */
 
 #else /* !defined(UKERNEL) */
--- openafs.orig/src/config/param.s390_linux26.h
+++ openafs/src/config/param.s390_linux26.h
@@ -86,6 +86,11 @@
 #endif
 #endif /* KERNEL */
 
+#ifdef __GLIBC__
+#if (__GLIBC__ > 2) || (__GLIBC__ == 2 && __GLIBC_MINOR__ > 3)
+#define USE_UCONTEXT
+#endif
+#endif
 #endif /* AFS_PARAM_H */
 
 #else /* !defined(UKERNEL) */
--- openafs.orig/src/config/param.s390x_linux26.h
+++ openafs/src/config/param.s390x_linux26.h
@@ -89,6 +89,11 @@
 #endif
 #endif /* KERNEL */
 
+#ifdef __GLIBC__
+#if (__GLIBC__ > 2) || (__GLIBC__ == 2 && __GLIBC_MINOR__ > 3)
+#define USE_UCONTEXT
+#endif
+#endif
 #endif /* AFS_PARAM_H */
 
 #else /* !defined(UKERNEL) */
